macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -204,7 +193,7 @@
 			);
 			name = Runner;
 			productName = Runner;
-			productReference = 33CC10ED2044A3C60003C045 /* Flutter Gallery.app */;
+			productReference = 33CC10ED2044A3C60003C045 /* gallery.app */;
```

Comment:
Comments should use `macos/Flutter/ephemeral/.app_filename`?

Reply:
No, I think in the comments Xcode actually hydrates the build settings, it thinks the name of the product is `gallery.app`.